### ISSUE TITLE
Doesn't work

### ISSUE THREAD

**Issue by diane-steiker** [kush789/Google-form-spammer#2]
I was trying this with a test form ([this actually](https://docs.google.com/forms/d/1KjlQtxlE3tw0_bxYPUkP239LyFRSY21IdUDhKeBBb7I/viewform)) on Ubuntu and it prints filled form but no response appears.

**Comment by nickzuck** [comment]
Yes you are correct. I tried it from both my local computer and ec2 server.. It fails badly.. On the other hand when I fill the form manually it doesn't work

**Comment by kush789** [comment]
Yes, this script no longer works. After material was used to redesign Google forms, all the control types (text, textarea etc.) which were used to fill the form were changed to type "HiddenControl". This makes it impossible (or so I believe) to distinguish between a text box and say, a select box. Hence, the script is failing. 

I haven't been able to find time to actually find another way to automate the filling (text is easy, however other options such as select, date etc. are tough to figure out if you don't know what kind of control it is in the first place).
